ddtrace/internal/gateway.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+from typing import Set
+from typing import TYPE_CHECKING
+
+import attr
+
+
+if TYPE_CHECKING:
+    from ddtrace.span import _RequestStore
+
+
+class _Addresses(Enum):
```

Comment:
Should these classes live in the appsec module in a `_gateway.py` module?

Reply:
No strong opinion for me. I will move them